app/src/backend/NotificationService.ts

```diff
@@ -0,0 +1,48 @@
+import { smtpOptions } from "@/lib/email";
+import nodemailer, { Transporter } from "nodemailer";
+
+interface EmailOptions {
+  to: string;
+  subject: string;
+  text: string;
+  html: string;
+}
+
+interface SendEmailResponse {
+  success: boolean;
+  messageId?: string;
+  error?: any;
+}
+
+class NotificationService {
+  private transporter: Transporter;
+  constructor() {
+    this.transporter = nodemailer.createTransport({ ...smtpOptions });
+  }
```

Comment:
No state in service. Create this in sendEmail or create static helper method in NotificationService.